patches/api/0495-Add-new-TickRateChangedEvent.patch

```diff
@@ -0,0 +1,72 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Axionize <154778082+Axionize@users.noreply.github.com>
+Date: Tue, 8 Oct 2024 20:38:01 -0400
+Subject: [PATCH] Add new TickRateChangedEvent
+
+
+diff --git a/src/main/java/org/bukkit/event/server/TickRateChangeEvent.java b/src/main/java/org/bukkit/event/server/TickRateChangeEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..97c13fe34ca28775c6757dc2aac28d8d7a375bdf
+--- /dev/null
++++ b/src/main/java/org/bukkit/event/server/TickRateChangeEvent.java
+@@ -0,0 +1,60 @@
++package org.bukkit.event.server;
++
++import org.bukkit.command.CommandSender;
++import org.bukkit.event.Cancellable;
++import org.bukkit.event.HandlerList;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * Fired when the server's target tickrate changes
++ */
++public class TickRateChangeEvent extends ServerEvent implements Cancellable {
```

Comment:
This event doesn't implement the static method getHandlerList so it just doesn't work at all and plugins that use it fail to load